org.omg.kerml.xtext/src/org/omg/kerml/xtext/util/KerMLRepositorySaveUtil.java

```diff
@@ -179,6 +180,9 @@ public void run(String[] args) {
 				this.initialize();				
 				this.read(args);
 				
+				System.out.println("Transforming...");
+				ElementUtil.transformAll(this.resourceSet, true);
```

Comment:
@seidewitz @manasbajaj If we don't want to upload the implicit generalizations, as you have written in the pull request description, I believe we should remove this update from the repository save utility, as it is responsible for adding the implicit generalizations to the model (although it is possible that it does not work as intended, as I did not manage to test this functionality in detail).

Reply:
Ah, yes, you are correct. I had been thinking of adding a command argument option on whether to add the implicit generalizations or not, but I forgot. Thanks for noticing this.

It would also be possible to add an option for this for the `%publish` command, of course, but that didn't seem like something our Jupyter users would particularly care about.